pkg/ipam/node_manager.go

```diff
@@ -181,9 +181,12 @@ func NewNodeManager(instancesAPI AllocationImplementation, k8sAPI CiliumNodeGett
 
 // Start kicks of the NodeManager by performing the initial state
 // synchronization and starting the background sync go routine
-func (n *NodeManager) Start(ctx context.Context) {
+func (n *NodeManager) Start(ctx context.Context) error {
 	// Trigger the initial resync in a blocking manner
-	n.instancesAPI.Resync(ctx)
+	resyncTime := n.instancesAPI.Resync(ctx)
```

Comment:
I assume the `Resync` call stack at some point it will log the error from the cloud provider? If yes maybe add in the lower `fmt.Errorf` something like `"please check previous logs for errors"`?